### DISCUSSION TITLE
求问Anime4k可以1.5倍这样放大吗？

### DISCUSSION THREAD

**Discussion by sao321** [Blinue/Magpie#825] [Q&A] [answered]
貌似Anime4k只可以2倍放大，或者加两个变成4倍放大。

求问有没有可能1.5放大吗？

感谢！

**Comment by Blinue** [comment]
你可以查看 Anime4K 的官方仓库 https://github.com/bloc97/Anime4K

大部分 Anime4K 着色器是两倍放大，有一些是 3 倍或 4 倍，但没有移植。Anime4K 不支持非整数倍缩放。

**Reply by sao321** [reply]
求问大佬如何更改为3倍，直接修改Anime4K_Upscale_UL.hlsl的
OUTPUT_WIDTH INPUT_WIDTH * 2
OUTPUT_HEIGHT INPUT_HEIGHT * 2
为3貌似不行...

主要原因是玩古董GAL时用2倍在4K屏幕上不够大（如图），用4倍又变成全屏了没地方放其它软件hook来的参考翻译...
![1](https://github.com/Blinue/Magpie/assets/59861512/52233f00-9791-45c3-ad44-37ba1056caa9)

**Reply by Blinue** [reply, marked as answer]
你可以在缩放模式的最后添加一个支持任意缩放倍数的后处理效果，比如 Bicubic、Lanczos
![image](https://github.com/Blinue/Magpie/assets/34770031/d321899c-6664-45bd-bd63-ede70cb686e0)

**Reply by sao321** [reply]
感谢！
